views/audit/README.md

```diff
@@ -0,0 +1,119 @@
+        Note: gcloud **alpha** is needed in order to use the parameter `--use-partitioned-tables` 
+    *   [Cloud Console Logs Viewer](https://cloud.google.com/logging/docs/export/configure_export_v2#dest-create)
+        Use this filter:
+        #### protoPayload.metadata.@type="type.googleapis.com/google.cloud.audit.BigQueryAuditMetadata" AND ( (protoPayload.metadata.jobChange.job.jobConfig.queryConfig.statementType="SCRIPT" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE" ) OR ( protoPayload.metadata.jobChange.job.jobStats.parentJobName!="" AND protoPayload.metadata.jobChange.job.jobStatus.jobState="DONE") OR protoPayload.metadata.tableDataChange.reason!="" OR protoPayload.metadata.tableDataRead.reason!=""  OR protoPayload.metadata.tableDeletion.reason!="" )
+        *   [Partitioning](https://cloud.google.com/logging/docs/export/bigquery#partition-tables)
+            is not required, but it is strongly recommended to select it for your BigQuery destination
+            
+    Note: You can create a log sink at the folder, billing account, or organization level using an 
+    [aggregated sink](https://cloud.google.com/logging/docs/export/aggregated_sinks#creating_an_aggregated_sink).
+1.  The log sink will immediately create the BigQuery dataset but the table will
+    be created once you run a BigQuery job post log sink creation.
+1.  To use the SELECT statement in
+    [big_query_elt_audit_log_v2.sql](/views/audit/big_query_elt_audit_log_v2.sql), change
+    all occurrences of
+    `project_id.dataset_id.cloudaudit_googleapis_com_data_access` to be the full
+    table path you created in step 1.
+    *   `sed
+        's/project_id.dataset_id.cloudaudit_googleapis_com_data_access/YOUR_PROJECT.YOUR_DATASET.YOUR_TABLE/'
+        big_query_elt_audit_log_v2.sql`
+1.  From here, you can do further analysis in BigQuery by querying the view, or
+    you can connect it to a BI tool such as DataStudio as a data source and
+    build dashboards.
+    
+#### Usage Examples
+Change all occurrences of `YOUR_VIEW` to the full path to the view. 
+
+* Run this query to see a detailed list of scripts which modified tables. The results are ordered with the most recent script runs first, and then further ordering is applied using the script's job id.
+  
+  
+  ```  
+  SELECT 
+   COALESCE(jobChange.jobStats.parentJobName, jobId) AS common_script_job_id,
+   jobChange.jobConfig.queryConfig.query,
+   jobChange.jobStats.queryStats.totalBilledBytes,
+   jobChange.jobConfig.queryConfig.statementType,
+   jobChange.jobStats.createTime,
+   jobChange.jobStats.startTime,
+   jobChange.jobStats.endTime,
+   jobRuntimeMs,
+   tableDataChange.deletedRowsCount,
+   tableDataChange.insertedRowsCount,
+  FROM YOUR_VIEW 
+  WHERE 
+  hasJobChangeEvent AND
+  hasTableDataChangeEvent AND
+  (jobChange.jobStats.parentJobName IS NOT NULL OR jobChange.jobConfig.queryConfig.statementType = 'SCRIPT')
+  ORDER BY 
+   jobChange.jobStats.startTime DESC,
```

Comment:
Remove DESC from ORDER BY

Reply:
Actually use this ordering instead:

ORDER BY 
 common_script_job_id,
 eventTimestamp DESC,
 jobChange.jobStats.startTime